demisto_sdk/commands/create_id_set/create_id_set.py

```diff
@@ -14,6 +21,54 @@ def __init__(self, output: str = '', input: str = '', print_logs: bool = True):
         self.output = output
         self.input = input
         self.print_logs = print_logs
+        self.id_set = OrderedDict()  # type: ignore
 
     def create_id_set(self):
-        return re_create_id_set(id_set_path=self.output, pack_to_create=self.input, print_logs=self.print_logs)
+        self.id_set = re_create_id_set(id_set_path=self.output, pack_to_create=self.input, print_logs=self.print_logs)
+        self.modify_id_set_command_to_integration_of_playbook()
+        return self.id_set
+
+    def modify_id_set_command_to_integration_of_playbook(self):
+        """
+        Modifies playbook set in id_set dictionary once it was created, and finally saves it to a file.
+        Each playbook that has "command_to_integration" field will be modified :
+        - command name value will be a list of all integrations that implements this command (instead of use "" ).
```

Comment:
Seems to me like we could be doing this more efficiently.
Could you explain why this is the most efficient way to achieve this in your opinion?

Reply:
Unfortunately, I can't think of a better and more efficient idea for doing it. My goal is that each command used in the playbook will be mapped to all integrations that implemented it. To do that, I have to go over all integrations set and create such a mapping. I cannot see other solution that does not cost in go over all integrations set to create this mapping and at the same time satisfies my goal. As I wrote in previous comments, we can discuss changing this modify of id_set. WDYT?